CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift

```diff
@@ -586,11 +561,12 @@ public let TYPE_NODES: [Node] = [
       Child(
         name: "scopedKeyword",
         kind: .token(choices: [.keyword(.scoped)]),
+        documentation: "scoped lifetime dependence specifier",
```

Comment:
Could you be a little more elaborate here, ie. what `scoped` actually does.

Reply:
Updated to : lifetime of return value is scoped to the lifetime of the original value